src/browser.mts

```diff
@@ -96,7 +96,11 @@ const PlaytBrowserClient = ({
     return AnybrainStopSDK();
   };
 
-  return { initialize, startMatch, stopMatch };
+  const reportError = async (error: any) => {
```

Comment:
I'd name it like this or so, so that it's obvious that it shouldn't be used for any odd uncritical error, because it will end the game for the user by redirecting them away or so
```suggestion
  const reportFatalError = async (error: unknown) => {
```